spec/product.sku.spec.tsx

```diff
@@ -1,23 +1,36 @@
-import React from "react"
-import { render, screen, waitFor } from "@testing-library/react"
+import React, { useState } from "react"
+import { fireEvent, render, screen, waitFor } from "@testing-library/react"
 import { NostoProvider, NostoPlacement, NostoProduct } from "../src/index"
 import RecommendationComponent from "./renderer"
 import "@testing-library/jest-dom"
 import { listenTo, WAIT_FOR_TIMEOUT } from "./utils"
 
-test("Product page with SKU id", async () => {
+function ProductPage() {
+  const productId = "7078777258043"
+  const [skuId, setSkuId] = useState("123")
+
   const tagging = {
-    product_id: "7078777258043",
-    selected_sku_id: "234"
+    product_id: productId,
+    selected_sku_id: skuId
   }
-  
+
+  return <>
+    <NostoPlacement id="productpage-nosto-1" />
+    <NostoProduct product={productId} tagging={tagging} />
+    <select data-testid="select" value="123" onChange={e => setSkuId(e.target.value)}>
```

Comment:
Also add test for multi-level variants? Like size, colors available in that size etc...?

Reply:
fair enough, I can try to replace with e.g. color and size pickers where the sku id would be color/size